test/unit/test_http_retries.py

```diff
@@ -40,6 +41,7 @@ def test_create_detector_attempts_retries(gl: Groundlight):
     )
```

Comment:
^ This test is pretty weird, its attempting to create a detector w/ an existing name, which will fail every time on the backend. Then its overwriting what the backend returns w/ its own error codes in `run_test`.